docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -309,5 +309,99 @@ Both visualizations are now displayed on your custom dashboard.
 IMPORTANT: By default, Discover shows data for the last 15 minutes. If you have a time-based index
 and no data displays, you might need to increase the time range.
 
+[[open-telemetry-aws-lambda-elastic]]
+==== AWS Lambda Support
+
+AWS Lambda functions can be instrumented with OpenTelemetry and monitored with Elastic Observability.
+
+To get started, follow the official AWS Distro for OpenTelemetry Lambda https://aws-otel.github.io/docs/getting-started/lambda[getting started documentation] and configure the OpenTelemetry Collector to output traces and metrics to your Elastic cluster.
+
+[[open-telemetry-aws-lambda-elastic-java]]
+==== Instrumenting AWS Lambda Java functions
+
+Note that for better startup time, it is recommended to use SDK based instrumentation (aka manual instrumentation of the code) rather than auto instrumentation.
```

Comment:
```suggestion
NOTE: For a better startup time, we recommend using SDK-based instrumentation, i.e. manual instrumentation of the code, rather than auto instrumentation.
```